### DISCUSSION TITLE
Welcome to Hierarchical-Multi-Label-Text-Classification Discussions!

### DISCUSSION THREAD

**Discussion by RandolphVI** [RandolphVI/Hierarchical-Multi-Label-Text-Classification#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by StefanRaab** [comment]
Hey @RandolphVI  I am currently working on your paper and I really like your approach! I have one question open: In the evaluation task, did you use the input from the Word Embeddings also for your Baseline or did you choose other formats for text representation?

**Reply by RandolphVI** [reply]
@StefanRaab  Our approach and the baseline all use the same Word Embeddings for the fairness.

**Reply by StefanRaab** [reply]
@RandolphVI  thank you for that fast reply!
